tree/dataframe/src/RLoopManager.cxx

```diff
@@ -692,3 +692,13 @@ const ColumnNames_t &RLoopManager::GetBranchNames()
    }
    return fValidBranchNames;
 }
+
+bool RLoopManager::HasDSValuePtrs(const std::string &col) const
+{
+   return fDSValuePtrMap.find(col) != fDSValuePtrMap.end();
+}
+
+void RLoopManager::AddDSValuePtrs(const std::string &col, const std::vector<void *> ptrs)
```

Comment:
Why is this a vector BTW? Is this for types that consist of multiple columns of primitive types?

Reply:
it's one vector element per processing slot (~per thread).